torch/_inductor/scheduler.py

```diff
@@ -1524,7 +1547,7 @@ def found_path(node):
                 else:
                     # continue DFS of new ancestors introduced by the fusion
                     return bool(combined_names & node.ancestors) or any(
-                        found_path(self.name_to_fused_node[n])
+                        found_path(self.name_to_fused_node[n])  # type: ignore[index]
```

Comment:
why is this necessary?

Reply:
mypy ignores the assertion added at line 1530. It thinks `self.name_to_fused_node` can be None.